src/diem/testing/miniwallet/client.py

```diff
@@ -60,15 +60,15 @@ def get(self, path: str) -> Dict[str, Any]:
 
     def send(self, method: str, path: str, data: Optional[str] = None) -> requests.Response:
         url = "%s/%s" % (self.server_url.rstrip("/"), path.lstrip("/"))
-        self.logger.info("%s %s: %s" % (method, path, data))
+        self.logger.debug("%s %s: %s" % (method, path, data))
         resp = self.session.request(
             method=method,
             url=url.lower(),
             data=data,
             headers={"Content-Type": "application/json", "User-Agent": jsonrpc.client.USER_AGENT_HTTP_HEADER},
         )
-        self.logger.info("response status code: %s" % resp.status_code)
-        self.logger.info("response body: %s" % resp.text)
+        self.logger.debug("response status code: %s" % resp.status_code)
```

Comment:
why sometimes use `debug("something: %s", my_var)` and sometimes `debug("something: %s" % my_var)` ?

Reply:
updated. it is because I learned new way to log without substitute message string.
so old code is all doing `debug("something: %s" % my_var)`, new code use `debug("something: %s", my_var)`, but I've changed everything under miniwallet to use new style. other places maybe update when it is touched.